### DISCUSSION TITLE
New line after ( in subquery

### DISCUSSION THREAD

**Discussion by license2kill** [andialbrecht/sqlparse#761] [Q&A]
Dear all,

1,
My invoke program is
"
print(sqlparse.format(first, reindent=True, keyword_case='upper', ofrmt='python', wrap_after=8, indent_after_first=True))
".

2,
How can I make the result

"
  SELECT dt,
       count(1)
  FROM
    (SELECT dt,
            WEEK
     FROM fin_dim.dim_m_y23_sixth_weekday_jwm_a)
  GROUP BY 1;
"

to the below

"
  SELECT dt,
               count(1)
  FROM
    (
     SELECT dt,
            WEEK
     FROM fin_dim.dim_m_y23_sixth_weekday_jwm_a
    )
  GROUP BY 1;
".

3,
ie. I want to change the format of subquery without changing "count(1)".

Your reply is appreciated.

**Comment by andialbrecht** [comment]
Unfortunately that's not possible right now. Feel free to file a feature request.
